docs/website/docs/user-guides/advanced/debugging-with-openshift-toolkit.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Debugging an Application with OpenShift Toolkit
+sidebar_position: 7
+---
+
+Debugging is an unavoidable part of development, and it can prove even more difficult when developing an application that runs remotely.
+
+However, this task is made absurdly simple with the help of the OpenShift Toolkit IDE plugin.
+
+## OpenShift Toolkit
+OpenShift Toolkit is an IDE plugin available on [VS Code](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector) and [Jet Brains](https://plugins.jetbrains.com/plugin/12030-openshift-toolkit-by-red-hat/) IDEs, that allows you to do all things that `odo` does, i.e. create, test, debug and deploy cloud-native applications on a cloud-native environment in simple steps.
+`odo` enables this plugin to do what it does.
+
+## Prerequisites
+1. [You have logged in to your cluster](../quickstart/nodejs.md#step-1-connect-to-your-cluster-and-create-a-new-namespace-or-project).
+2. You have [initialized an application with `odo`](/docs/command-reference/init), for example [the Node.JS quickstart application](../quickstart/nodejs.md#step-2-initializing-your-application-odo-init).
+:::note
+ This tutorial uses a Node.js application, but you can use any application that has Devfile with debug command defined in it. If your Devfile does not contain a debug command, refer to [Configure Devfile to support debugging](#configure-devfile-to-support-debugging).
+:::
+3. You have [installed](/docs/overview/installation#ide-installation) the OpenShift Toolkit Plugin in your preferred VS Code or a JetBrains IDE.
```

Comment:
As a prerequisite, you are saying that we can install the plugin on "JetBrains IDE", but the tutorial is focusing on VSCode. Maybe you can add a note mentioning that this tutorial uses VS Code, but there are similar steps with JetBrains IDEs"..